.github/workflows/main.yml

```diff
@@ -42,8 +42,8 @@ jobs:
           sudo apt-get -y update
           sudo apt-get -y install build-essential git cmake wget ninja-build
           cd emsdk
-          ./emsdk install 3.1.5
-          ./emsdk activate 3.1.5
+          ./emsdk install latest
+          ./emsdk activate latest
```

Comment:
I'm tempted to say that we should freeze versions and only update them when needed.

It is already an issue with our rust code: we don't push the `Cargo.lock` file which I think is a mistake: basically it makes CI jobs non-reproducible since on the same commit two attempts might use different minor version of the dependencies. Additionally, that's also why the rust cache is not working well on the CI workflow.

Reply:
Froze it to 3.1.6 instead of latest. Created #173 for the Cargo.lock issue